{
    "config": {
        "abort": {
            "already_configured": "Za\u0159\u00edzen\u00ed je ji\u017e nastaveno",
            "cannot_connect": "Nelze se p\u0159ipojit k objeven\u00e9mu za\u0159\u00edzen\u00ed Volumio"
        },
        "error": {
            "cannot_connect": "Nepoda\u0159ilo se p\u0159ipojit",
            "unknown": "Neo\u010dek\u00e1van\u00e1 chyba"
        },
        "step": {
            "discovery_confirm": {
                "description": "Chcete p\u0159idat Volumio (`{name}`) do Home Assistant?",
                "title": "Objeven\u00e9 instance Volumio"
            },
            "user": {
                "data": {
                    "host": "Hostitel",
                    "port": "Port"
                }
            }
        }
    }
}